package.json

```diff
@@ -11,8 +11,8 @@
   "devDependencies": {
     "mocha": "^2.3.4"
   },
-  "repository" : {
-    "type" : "git",
-    "url" : "https://github.com/rolling-scopes-school/js-assignments.git"
+  "repository": {
```

Comment:
the PR has redundant changes in styling, spaces, comments....
Please follow the simple rules in the future:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR